app/controllers/admins/users_controller.rb

```diff
@@ -19,7 +19,16 @@ def index
     end
 
     def update
-      if user.update(user_params)
+      updated = if user_params[:password].blank? &&
```

Comment:
why we can't check only `password` ?

Reply:
If we check only passwor, password confirmation will be skipped